src/main/java/com/synopsys/integration/detect/workflow/codelocation/CodeLocationNameGenerator.java

```diff
@@ -104,6 +104,16 @@ public String createBinaryScanCodeLocationName(File targetFile, String projectNa
         return createCodeLocationName(prefix, fileCodeLocationNamePieces, suffix, fileCodeLocationEndPieces);
     }
 
+    public String createContainerScanCodeLocationName(File targetFile, String projectName, String projectVersionName) {
```

Comment:
There is quite a bit of code duplication in this file. Not necessarily as part of this PR but it might be good to consolidate it.

Reply:
Yup that's something I did notice but continued with the existing pattern. Feel free to create a ticket to address this as a clean-up tech debt - certainly would make this framework cleaner.